doc/api/stream.md

```diff
@@ -1590,6 +1590,10 @@ async function run() {
 run().catch(console.error);
 ```
 
+`stream.pipeline()` will call `stream.destroy(err)` on all streams except:
+- `Readable` streams which have emitted `'end'` or `'close'`.
+- `Writable` streams which have emitted `'finish'` or `'close'`.
```

Comment:
@mcollina: I'm not sure what the rationale of this behaviour is. Maybe we could add that?

Reply:
New-ish markdown lint rule will enforce `*` for lists instead of `-`.